Lib/sndhdr.py

```diff
@@ -158,10 +157,10 @@ def test_wav(h, f):
     # 'RIFF' <len> 'WAVE' 'fmt ' <len>
     if not h.startswith(b'RIFF') or h[8:12] != b'WAVE' or h[12:16] != b'fmt ':
         return None
-    f.seek(0)
     try:
+        f.seek(0)
```

Comment:
Why this line has been moved?

Reply:
This didn't need to be moved, that was my mistake!